README.md

```diff
@@ -211,32 +211,113 @@ This can be useful if using a column for doc references.
+The CSV example below using the reference will find the `doc` with `district_1` and create a `parent` property with the value of the `district_1` doc
+
 To reference other docs, replace the type suffix with a matching clause:
 
-	GET location:place WHERE external_id=COL_VAL
+	parent:place WHERE reference_id=COL_VAL
+
+Refered to CSV Example:
+
+| reference_id:excluded | parent | is_name_generated | name | external_id | notes | geolocation | reported_date:timestamp |
+| --------------------- | ------ | ----------------- | ---- | ----------- | ----- | ----------- | ----------------------- |
+| district_1            |        | false             | D1   |             |       |             | 1544031155715           |
+| district_2            |        | false             | D2   |             |       |             | 1544031155715           |
+| district_3            |        | false             | D3   |             |       |             | 1544031155715           |
+
+CSV Using Reference 
+
+| reference_id:excluded | parent:place WHERE reference_id=COL_VAL | is_name_generated | name | external_id | notes | geolocation | reported_date:timestamp |
+| --------------------- | --------------------------------------- | ----------------- | ---- | ----------- | ----- | ----------- | ----------------------- |
+| health_center_1       | district_1                              | false             | HC1  |             |       |             | 1544031155715           |
+| health_center_2       | district_1                              | false             | HC2  |             |       |             | 1544031155715           |
+| health_center_3       | district_1                              | false             | HC3  |             |       |             | 1544031155715           |
+| health_center_4       | district_1                              | false             | HC4  |             |       |             | 1544031155715           |
+| health_center_5       | district_1                              | false             | HC5  |             |       |             | 1544031155715           |
+| health_center_6       | district_2                              | false             | HC6  |             |       |             | 1544031155715           |
+| health_center_7       | district_2                              | false             | HC7  |             |       |             | 1544031155715           |
+| health_center_8       | district_2                              | false             | HC8  |             |       |             | 1544031155715           |
+| health_center_9       | district_2                              | false             | HC9  |             |       |             | 1544031155715           |
+| health_center_10      | district_2                              | false             | HC10 |             |       |             | 1544031155715           |
```

Comment:
It would be sufficient to have 3-4 lines here. As long as we show a couple with different district, and a couple with the same, the message would get across.